config/locales/en.yml

```diff
@@ -572,9 +572,7 @@ en:
     apicast: APIcast
     apicast_hint: Nginx reverse proxy server.
 
-    service_mesh_istio: |
-      Istio
-      <span>A uniform way to secure, connect, and monitor microservices.</span>
+    service_mesh_istio: Istio
```

Comment:
can we also remove the problematic css which hid it in the past?

Reply:
I also see a lot more HTML in this file. Can we figure out which items are for hints which tolerate it and which are for normal tips that escape it and make it visible by users?

Reply:
True, this `label span` [here](https://github.com/3scale/porta/blob/fix-unwanted-span-at-product-settings-istio/app/javascript/src/Settings/styles/authentication.scss#L6) doesn't seem to be needed anymore (while `inline-hints` is still required, unless we want to remove [this hint](https://github.com/3scale/porta/blob/fix-unwanted-span-at-product-settings-istio/config/locales/en.yml#L1262) also.

By the way, looking at that CSS, it seems VERY weird that "3scale managed" is added via CSS, and not through i18n strings.

Reply:
I've decided to remove the file and fix that weird CSS tag "3scale managed" including it inside the en.yml. 

About the HTMLs, I've been checking them and have found some problems with some parts of it. Everything I checked was working as expected, but I'll probably create a new issue to dig deeper.